gph/src/ripser.h

```diff
@@ -92,7 +93,7 @@ typedef uint16_t coefficient_t;
 
 using barcodes_t = std::vector<value_t>;
 
-static const size_t num_coefficient_bits = 8;
+const size_t num_coefficient_bits = 8;
```

Comment:
In general, when do we want const and when do we want static const?

Reply:
Good question.
When using static, the variable can only be accessed within the file itself, or compilation unit.
It could be similar to a private attribute for a Class, the static variable is can only be directly accessed within the file where it is declared.

Reply:
Thanks!